packages/tokamak/sdk/package.json

```diff
@@ -67,7 +67,8 @@
     "@tokamak-network/core-utils": "workspace:*",
     "lodash": "^4.17.21",
     "merkletreejs": "^0.3.10",
-    "rlp": "^2.2.7"
+    "rlp": "^2.2.7",
+    "semver": "^7.6.0"
```

Comment:
Where do we use it?

Reply:
OP team used [here](https://github.com/tokamak-network/tokamak-thanos/pull/181/files#diff-21be87624c7a8a8ca2cee114f74f8846c0b99fc2e35466f0451175659111010fR266)